sdk/build.gradle

```diff
@@ -48,4 +54,33 @@ task bumpAndTag {
     doLast {
         LibraryProject.INSTANCE.tagAndIncrement(LibraryProject.INSTANCE.getLibraryVersionCode() + 1)
     }
+}
+
+tasks.withType(Javadoc).all {
+    /* Disable javadoc since we do not have any java file */
+    enabled = false
```

Comment:
Can you remove the javadoc tasks alltogether ?  If bintray complains, we can upload an empty javadoc artifiact. Some other projects do this.